.github/workflows/npm-release.yml

```diff
@@ -0,0 +1,36 @@
+name: Release to NPM
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  release:
+    name: Release to NPM and github
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v1
+        with:
+          fetch-depth: 1
+      - name: Setup Node.js
+        uses: actions/setup-node@v1
+        with:
+          node-version: 12
+      - name: Cache dependencies
+        uses: actions/cache@v1
+        with:
+          key: npm-${{ hashFiles(format('{0}{1}', github.workspace,
+            '/package-lock.json')) }}
+          path: ~/.npm
+          restore-keys: |
+            npm-
+      - name: Install dependencies
+        run: npm ci --ignore-scripts --no-audit --prefer-offline --progress=false
+      - name: Release
```

Comment:
I think we are missing here the script to match the versions. `scripts/release.js` This is to make sure that when running `npx poetic` it will download the correct version of the package. Ideas about this?

Reply:
`semantic-release` will bump the version if this is a concern?  Or I missing why `scripts/release.js` exists :) ?

Reply:
I thought of a workaround that is better than the current approach, so don't worry about this. Thanks